access_qc_aggregator/access_qc_aggregator.cwl

```diff
@@ -0,0 +1,554 @@
+    'sbg:x': 0
+    'sbg:y': 1175.453125
+  - id: duplex_bam_stats_pool_b_dir
+    type:
+      type: array
+      items:
+        - File
+        - Directory
+        - 'null'
+    label: duplex_bam_stats_pool_b_dir
+    'sbg:x': 0
+    'sbg:y': 1068.59375
+  - id: collapsed_bam_stats_pool_a_dir
+    type:
+      type: array
+      items:
+        - File
+        - Directory
+        - 'null'
+    label: collapsed_bam_stats_pool_a_dir
+    'sbg:x': 0
+    'sbg:y': 2030.328125
+  - id: collapsed_bam_stats_pool_b_dir
+    type:
+      type: array
+      items:
+        - File
+        - Directory
+        - 'null'
+    label: collapsed_bam_stats_pool_b_dir
+    'sbg:x': 0
+    'sbg:y': 1923.46875
+  - id: collapsed_bam_duplex_metrics_pool_a_dir
+    type:
+      type: array
+      items:
+        - File
+        - Directory
+        - 'null'
+    label: collapsed_bam_duplex_metrics_pool_a_dir
+    'sbg:x': 0
+    'sbg:y': 2244.046875
+  - id: collapsed_bam_duplex_metrics_pool_b_dir
+    type:
+      type: array
+      items:
+        - File
+        - Directory
+        - 'null'
+    label: collapsed_bam_duplex_metrics_pool_b_dir
+    'sbg:x': 0
+    'sbg:y': 2137.1875
+  - id: gatk_mean_quality_by_cycle_recal_dir
+    type:
+      type: array
+      items:
+        - File
+        - Directory
+        - 'null'
+    label: gatk_mean_quality_by_cycle_recal_dir
+    'sbg:x': 0
+    'sbg:y': 320.578125
+  - id: gatk_mean_quality_by_cycle_dir
+    type:
+      type: array
+      items:
+        - File
+        - Directory
+        - 'null'
+    label: gatk_mean_quality_by_cycle_dir
+    'sbg:x': 0
+    'sbg:y': 427.4375
+  - id: uncollapsed_bam_stats_pool_b_dir
+    type:
+      type: array
+      items:
+        - File
+        - Directory
+        - 'null'
+    label: uncollapsed_bam_stats_pool_b_dir
+    'sbg:x': 0
+    'sbg:y': 0
+  - id: uncollapsed_bam_stats_pool_a_dir
+    type:
+      type: array
+      items:
+        - File
+        - Directory
+        - 'null'
+    label: uncollapsed_bam_stats_pool_a_dir
+    'sbg:x': 0
+    'sbg:y': 106.859375
+  - id: biometrics_threads
+    type: int?
+    label: biometrics_threads
+    doc: Number of threads to use.
+    'sbg:x': 0
+    'sbg:y': 2350.90625
+  - id: duplex_biometrics_coverage_threshold
+    type: int?
+    label: duplex_biometrics_coverage_threshold
+    doc: Samples with Y chromosome above this value will be considered male.
+    'sbg:x': 0
+    'sbg:y': 961.734375
+  - id: duplex_biometrics_major_threshold
+    type: float?
+    label: duplex_biometrics_major_threshold
+    doc: Major contamination threshold for bad sample.
+    'sbg:x': 0
+    'sbg:y': 748.015625
+  - id: duplex_biometrics_minor_threshold
+    type: float?
+    label: duplex_biometrics_minor_threshold
+    doc: Minor contamination threshold for bad sample.
+    'sbg:x': 0
+    'sbg:y': 641.15625
+  - id: collapsed_extraction_files
+    type:
+      type: array
+      items: File
+      inputBinding:
```

Comment:
similarly to the other input in this workflow, `inputBinding` is not supported here unfortunately. The packed version will error if this gets included

Reply:
see my above comment